src/test/java/com/blackwaterpragmatic/webservices/resource/UserResourceTest.java

```diff
@@ -147,4 +131,163 @@ public void should_not_get_missing_user() throws URISyntaxException, JsonParseEx
 		assertEquals(expectedResponse, response.getContentAsString());
 	}
 
+
```

Comment:
I think some of this could be pulled out to common methods & into @Before, but I didn't know if you guys like the verbosity?